common/model/opening-hours.ts

```diff
@@ -0,0 +1,65 @@
+import { Moment } from 'moment';
+import { ImageType } from './image';
+import { Link as LinkType } from './link';
+
+export type Day = string; // 'Monday' | 'Tuesday' | 'Wednesday' | 'Thursday' | 'Friday' | 'Saturday' | 'Sunday';
+
+export type OverrideType =
+  | 'Bank holiday'
+  | 'Easter'
+  | 'Christmas and New Year'
+  | 'Late Spectacluar'
+  | 'other';
+
+export type OverrideDate = {
+  overrideDate: Moment,
+  overrideType: OverrideType,
```

Comment:
[This should be nullable](https://github.com/wellcomecollection/wellcomecollection.org/blob/master/common/model/opening-hours.js#L17).

Flow lets you have optional keys and nullable values based on which side of the colon the question mark goes. TypeScript only has optional keys (question mark before the colon), so to make something nullable, you have to declare null as a possible value, e.g.

`overrideType: OverrideType | null`

Looks like there are a few other places this'll need updating in this file.

Reply:
Thank you !